examples/demo.js

```diff
@@ -116,11 +117,15 @@ window.onload = function() {
       $(step.ui.querySelector('img')).show();
 
       step.imgElement.src = step.output;
+      step.linkElement.href = step.output;
+      step.linkElement.download = step.name + ".png"
```

Comment:
I think we should make this something like step.name + step.output.split('/')[1].split(';')[0];
This way even in we output jpeg or jpg files in the future they will get downloaded in the right format

Reply:
And perhaps we should break it out into a standalone function, like `function fileExtension(output)` -- thank you!! It could be defined just above this line. 

This is exciting!

Reply:
@jywarren yeah that sound cool!!